201-vm-os-disk-and-data-disk-existing-vnet/azuredeploy.json

```diff
@@ -42,12 +42,6 @@
         "description": "Name of the existing VNET"
       }
     },
-    "existingVirtualNetworkResourceGroup": {
```

Comment:
This is still needed in the subnet variable - should be the first parameter of the resourceId function

Reply:
Added back existingVirtualNetworkResourceGroup parameter and updated at subnetref